tests/compile/callingconvention.srctree

```diff
@@ -35,10 +35,19 @@ p2 = f2
 p3 = f3
 p4 = f4
 
+
+######## callingconvention.h ########
+
+#define DLL_EXPORT
+#include "external_callingconvention.h"
+
+
 ######## external_callingconvention.h ########
 
 #ifndef DL_IMPORT
   #define DL_IMPORT(t) t
+#elif defined(DLL_EXPORT)
+  #define DL_IMPORT(t) DL_EXPORT(t)
```

Comment:
The way I've rewritten this, it might be clearer if `callingconvention,.h` were named `callingconvention_internal.h`, in that it is meant only for use when compiling the module, not when linking to it.